entity-service-impl/src/main/java/org/hypertrace/entity/query/service/converter/response/getter/ListValueGetter.java

```diff
@@ -8,16 +8,21 @@
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
 import com.google.inject.name.Named;
+import java.util.List;
 import org.hypertrace.entity.query.service.converter.ConversionException;
 import org.hypertrace.entity.query.service.v1.Value;
 
 @Singleton
 public class ListValueGetter implements ValueGetter {
   private final ValueGetter arrayGetter;
+  private final List<ValueGetter> rootGetters;
 
   @Inject
-  public ListValueGetter(@Named("array") final ValueGetter arrayGetter) {
+  public ListValueGetter(
+      @Named("array") final ValueGetter arrayGetter,
+      @Named("root_getters") final List<ValueGetter> rootGetters) {
     this.arrayGetter = arrayGetter;
+    this.rootGetters = rootGetters;
```

Comment:
Wondering if we could include the `arrayGetter` in the `rootGetters`!
Could you explore this?

Reply:
arrayGetter is always called in ListValueGetter.  ListValueGetter firstly always checks if node matches with `VALUES_LIST_KEY` and then we check if the node is an array, so I think we don't need to add arrayGetters in rootGetters as, array will always come inside the `valuesList` and arrayGetters.match() won't match anything in the ValueMapper.

Reply:
Got it! Thanks for checking this.